R/check-name.R

```diff
@@ -0,0 +1,25 @@
+#' Checks that the name of the study or file is valid.
+#' Valid strings contain: letters, numbers, spaces, underscores,
+#' hyphens, periods, plus signs, and parentheses.
+#'
```

Comment:
What does this do?

Reply:
Marking the function as internal removes the function from the package index. It's not too important, but it's something that pkgdown was complaining about when I built the site locally, because we have functions that are documented but not listed on the Reference page (because they're not exported).

Reply:
Ah, okay. I wanted to ask before I committed the suggestion in case I forgot about it later. Haha. Thanks for the explanation!